doc/source/ray-air/user-guides.rst

```diff
@@ -67,7 +67,7 @@ AIR Feature Guides
     +++
     .. link-button:: /ray-air/examples/serving_guide
         :type: ref
-        :text: How to configure AIR ModelWrapper for HTTP input or batching?
+        :text: How to configure AIR Predictor for HTTP input or batching?
```

Comment:
```suggestion
        :text: How to configure AIR PredictorDeployment for HTTP input or batching?
```

I'm not sure about this– should it be `PredictorDeployment` or `Predictor`?

Reply:
yeah i think deployment here make sense